src/common/hooks/useHandleOrderUpdateError/useHandleOrderUpdateError.js

```diff
@@ -32,6 +33,17 @@ const useHandleOrderUpdateError = (mutatorExpenseClass) => {
             values,
           });
         }
+      } else if (errorCode === ERROR_CODES.fundLocationRestrictionViolation) {
```

Comment:
So, in case there are several invalid associations between funds and locations it will display only the first?

Reply:
I have updated

https://github.com/folio-org/ui-orders/assets/116072773/f452a794-26af-4a28-866f-7b53110f7930